src/test/java/delete_event_use_case/DeleteEventControllerTest.java

```diff
@@ -0,0 +1,26 @@
+package delete_event_use_case;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+/**
+ * Unit Testing for DeleteEventController class
+ * @author : Khizer Ahmad
+ * @created: November 19th 2022
+ * @since : 2022-11-21, Mon.
+ **/
+public class DeleteEventControllerTest {
+    private final static DeleteEventPresenter outputBoundary = new DeleteEventPresenter();
+    private final static DeleteEventDataAccess dsGateway = new DeleteEventDataAccess();
+    private final static int dayIndex = 3;
+    private final static String eventTitle = "Event";
```

Comment:
I believe it would be better to initialize variables in the unit test as each unit test needs to avoid using static and as well keeping variables unit test specific.